pipelines/matrix/conf/base/disease_list/catalog.yml

```
@@ -0,0 +1,167 @@
+_layer_raw: &_layer_raw
+  metadata:
+    kedro-viz:
+      layer: raw
+
+_layer_int: &_layer_int
+  metadata:
+    kedro-viz:
+      layer: integration
+
+_layer_prm: &_layer_prm
+  metadata:
+    kedro-viz:
+      layer: primary
+
+###################################
+########### INPUT DATA ############
+###################################
+
+# DEPRECATED: No longer used - PyOxigraph Store is used instead (via mondo_graph)
+# disease_list.raw.mondo_owl:
+#   type: text.TextDataset
+#   <<: *_layer_raw
+#   filepath: http://purl.obolibrary.org/obo/mondo.owl
+
+disease_list.raw.mondo_graph:
+  type: matrix.datasets.rdf.PyOxiGraphDataset
+  <<: *_layer_raw
+  filepath: http://purl.obolibrary.org/obo/mondo.owl
+  load_args:
+    format: xml
+
+disease_list.raw.mondo_sssom:
+  type: pandas.CSVDataset
+  <<: *_layer_raw
+  filepath: http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv
+  load_args:
+    sep: "\t"
+    comment: "#"
+
+# @JacquesVergine I only want to download this file, its the start and end point
+# Does it need to be mentioned in the pipeline at all?
+disease_list.raw.mondo_obsoletion_candidates:
+  type: pandas.CSVDataset
+  <<: *_layer_raw
+  filepath: https://raw.githubusercontent.com/monarch-initiative/mondo/refs/heads/master/src/ontology/reports/mondo_obsoletioncandidates.tsv
+  load_args:
+    sep: "\t"
+
+disease_list.raw.icd10_cm_codes:
+  type: pandas.ExcelDataset
+  <<: *_layer_raw
+  filepath: https://www.cms.gov/files/document/valid-icd-10-list.xlsx
+  load_args:
+    sheet_name: 0
+
+#############################################
+########### Intermediate Outputs ############
+#############################################
+
+#TODO: Do intermediate outputs need to be saved to disk? Or can they be kept in memory only?
+
+disease_list.int.mondo_preprocessed:
+  type: matrix.datasets.rdf.PyOxiGraphDataset
+  <<: *_layer_int
+  filepath: ${globals:paths.tmp}/disease_list/mondo_preprocessed.owl
+  save_args:
+    format: xml
+
+# DEPRECATED: mondo_labels is no longer persisted (handled in-memory in merged node)
+# disease_list.int.mondo_labels:
+#   type: pandas.CSVDataset
+#   <<: *_layer_int
+#   filepath: ${globals:paths.tmp}/disease_list/mondo_labels.tsv
+
+disease_list.int.billable_icd10_codes:
+  type: pandas.CSVDataset
+  <<: *_layer_int
+  filepath: ${globals:paths.tmp}/disease_list/icd10_cm_billable_template.tsv
+  load_args:
+    sep: "\t"
+  save_args:
+    sep: "\t"
+    index: false
```

Can we move to parquet for intermediary files? It causes a lot of issues especially with string columns. It's as easy as this:
```suggestion
disease_list.int.billable_icd10_codes:
  type: pandas.ParquetDataset
  <<: *_layer_int
  filepath: ${globals:paths.tmp}/disease_list/icd10_cm_billable_template.parquet
```